### PR TITLE
Add poky bbappends to genivi-dev-platform

### PR DESCRIPTION
**Pull request by DunderRoffe**
Add the poky bbappends to genvi-dev-platform again. The layer config
was missed while restructuring bbappends in meta-genivi-dev.

Signed-off-by: Viktor Sjölind <viktor.sjolind@pelagicore.com>

### CODE REVIEW

File: meta-genivi-dev/poky/conf/layer.conf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH =. "${LAYERDIR}:"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/meta/recipes-*/*/*.bb \
+            ${LAYERDIR}/meta/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "genividev"
```
Comment (chbae) [inline meta-genivi-dev/poky/conf/layer.conf]: "genividev" -> "genividev-poky-append"

File: meta-genivi-dev/poky/conf/layer.conf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+BBFILE_PATTERN_genividev = "^${LAYERDIR}/"
```
Comment (chbae) [inline meta-genivi-dev/poky/conf/layer.conf]: "genividev" -> "genividev-poky-append"

File: meta-genivi-dev/poky/conf/layer.conf [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+BBFILE_PRIORITY_genividev = "7"
```
Comment (chbae) [inline meta-genivi-dev/poky/conf/layer.conf]: "genividev" -> "genividev-poky-append"

File: meta-genivi-dev/poky/conf/layer.conf [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+# This should only be incremented on significant changes that will
+# cause compatibility issues with other layers
+LAYERVERSION_genividev = "1"
```
Comment (chbae) [inline meta-genivi-dev/poky/conf/layer.conf]: "genividev" -> "genividev-poky-append"

Comment (chbae) [inline meta-genivi-dev/poky/conf/layer.conf]: Add

LAYERDEPENDS_genividev-poky-append = "genividev"

**Comment by DunderRoffe** [comment]
Thank you for the review @chbae. I belive I fixed all the things you noticed would you have another look?

**Pull request merged**